stylesheets/partials/base/_secondaryFooter.scss

```diff
@@ -2,14 +2,15 @@
 	background: $red;
 	color: $white;
 	padding: 20px 0;
-	
```

Comment:
Stupid question, but is this change per formatting standard?  Or, did the text editor do this and you decided not to fight it?

Reply:
I put it in there because it didn't look right to me. We aren't super strict on our formatting yet for sass.